content/zh/docs/tasks/traffic-management/ingress/kubernetes-ingress/index.md

```diff
@@ -15,13 +15,13 @@ test: yes
 而不是 Ingress 来利用 Istio 提供的完整功能集，例如丰富的流量管理和安全功能。
 {{< /tip >}}
 
-## 准备工作{#before-you-begin}
+## 准备工作  {#before-you-begin}
 
 请按照[入口网关任务](/zh/docs/tasks/traffic-management/ingress/ingress-control/)中的
 [准备工作](/zh/docs/tasks/traffic-management/ingress/ingress-control/#before-you-begin)、
 [确定 Ingress IP 和端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)的说明进行操作。
 
-## 使用 Ingress 资源配置入口网关 {#configuring-ingress-using-an-ingress-resource}
+## 使用 Ingress 资源配置入口网关  {#configuring-ingress-using-an-ingress-resource}
```

Comment:
```suggestion
## 使用 Ingress 资源配置入口网关 {#configuring-ingress-using-an-ingress-resource}
```
One space is ok and consistent with other pages. Not necessary to add one more again. Same below.